cookbook/configuration/web_server_configuration.rst

```diff
@@ -48,6 +48,12 @@ are:
             Allow from All
         </Directory>
 
+        # uncomment the following lines if you install assets as symlinks
+        # or run into problems when compiling LESS/Sass/CoffeScript assets
+        # <Directory /var/www/project>
```

Comment:
Good point! When/if this change is validated for the Symfony docs, I'll add it to the `.htaccess` file of the Standard Edition. Thanks.